README.md

```diff
@@ -1,62 +1,86 @@
 # cookiecutter-awesome
-Cookiecutter to create an [awesome][awesome] list.
+Cookiecutter to create an [awesome][awesome-list] list.
 
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
 - [Features](#features)
 - [Requirements](#requirements)
 - [Usage](#usage)
-- [Documentation](#documentation)
+- [Example](#example)
 - [Development](#development)
-- [History](#history)
+  - [History](#history)
+  - [Community](#community)
+- [Other Awesome Lists](#other-awesome-lists)
 - [Credits](#credits)
 - [License](#license)
 
 ## Features
+
 - Creative Commons CC0 1.0 Universal License (as recommended in awesome's
-  [manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md)).
+  [manifesto][manifesto]).
 - Travis configuration with [awesome_bot][awesome_bot] check.
 - Changelog, code of conduct, contributing, and pull request template files.
 - Compliant with awesome's [PR requirements][pr-requirements].
 
 ## Requirements
+
 - [git](https://git-scm.com/downloads)
 - [python](https://www.python.org/downloads/)
 - [cookiecutter](https://github.com/audreyr/cookiecutter)
 - ([awesome_bot][awesome_bot])
 
 ## Usage
+
 ```bash
-cookiecutter gh:Pawamoy/cookiecutter-awesome
+cookiecutter gh:moodule/cookiecutter-awesome
```

Comment:
Well indeed, if we're to merge this, the URI should be `gh:pawamoy/cookiecutter-awesome` :wink:

Reply:
hehe you did review everything! you deserve the credit :D

Reply:
It's not really about credit, it's just that if we merge it, it appears on the `pawamoy/cookiecutter-awesome` repository, so the instructions must match the repo's name :slightly_smiling_face: